CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Change history
 ==============
 
+0.2.0-rc1
+=====
+
+*07, 28, 2022*
```

Comment:
This date should probably be changed to the date on which the release actually happens